tests_e2e/orchestrator/lib/agent_test_suite.py

```diff
@@ -450,7 +450,17 @@ def _setup_test_nodes(self, test_context: AgentTestContext) -> None:
             #
             log.info("Extracting %s on the test node", self._test_tools_tarball_path)
             command = f"tar xvf {target_path/self._test_tools_tarball_path.name}"
-            log.info("Remote command [%s] completed:\n%s", command, ssh_client.run_command(command))
+            try:
+                log.info("Remote command [%s] completed:\n%s", command, ssh_client.run_command(command))
+            except CommandError as e:
+                # Some distros do not have tar installed by default. Install it if needed.
+                if "tar: command not found" in e.stderr:
```

Comment:
you may want to check the exit code instead of the error message (command not found == 127), but we can always change that if needed